README.md

```diff
@@ -1,59 +1,33 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio
 
-> One paragraph statement about the project.
+> Creating my portfolio
 
-![screenshot](./app_screenshot.png)
+![screenshot](./pic3.png)
```

Comment:
- [ ] [OPTIONAL] Good readme provided. But your screenshot is not displaying as expected. I believe you will try to fix it.